src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_upd_routines.F90

```diff
@@ -3500,7 +3532,20 @@ subroutine cat_enkf_increments(                               &
     real, dimension(     N_catd)          :: tp1_ensavg
 
     type(obs_param_type)                  :: this_obs_param
-        
+   
+    !jpark50
+    real, dimension(N_catd, N_ens)        :: wesn_sum_old
+    real, dimension(N_catd, N_ens)        :: areasc_old 
+    real, parameter:: cpw   = 2065.22 !ice specific heat capacity @ 0C [J/kg/K]
```

Comment:
Refer to the value in the StieglitzSnow module.  See comments about "use" statements below.